internal/kv/minio/minio_kv.go

```diff
@@ -187,6 +233,23 @@ func (kv *MinIOKV) MultiLoad(keys []string) ([]string, error) {
 	return objectsValues, resultErr
 }
 
+func (kv *MinIOKV) MultiLoadBytes(keys []string) ([][]byte, error) {
+	var resultErr error
+	objectsValues := make([][]byte, 0, len(keys))
+
+	for _, key := range keys {
+		objectValue, err := kv.LoadBytes(key)
+		if err != nil {
+			if resultErr == nil {
+				resultErr = err
+			}
```

Comment:
Current `MultiLoadBytes` fails if any key `LoadBytes` fails. May just return here for shortcut.

Reply:
Then what about `MultiLoad()`? Make them both return when encounter the first error?

Reply:
> 

lgtm